package.json

```diff
@@ -17,36 +18,43 @@
   "bugs": {
     "url": "https://github.com/PiwikPRO/ngx-piwik-pro/issues"
   },
+  "engines": {
```

Comment:
Thought: I think that we should add it also to the `/projects/ngx-piwik-pro/package.json`